src/app/AppMain.vue

```diff
@@ -31,6 +32,16 @@ if (!import.meta.env.SSR) {
 	performance.measure('gj-shell-init', 'gj-start');
 }
 
+// We allow users to access the onboarding flow even after they've gone through
+// onboarding.
+//
+// In case they did that, or didn't fully complete their onboarding, clear out
+// their onboarding-start timestamp now so they're no longer considered to be
+// onboarding.
+//
+// TODO(quests) test this
+Onboarding.clearOnboardingStartTimestamp();
```

Comment:
can't be here, gotta put it in onMounted, otherwise you'll break SSR

Reply:
fixed 👍